pkg/api/store/cluster/cluster_store.go

```diff
@@ -228,7 +230,7 @@ func loadDataFromTemplate(clusterTemplateRevision *v3.ClusterTemplateRevision, d
 		}
 		val, err := builder.ConvertSimple(question.Type, answer, builder.Create)
 		if err != nil {
-			return nil, err
+			return nil, httperror.WrapAPIError(err, httperror.ServerError, "Error processing ClusterTemplate answers")
```

Comment:
nit: in some messages we use camelcase like clusterTemplateRevision, in some uppercase like ClusterTemplate.

Reply:
changing all to use camelcase